source/documentation/values/calculations.md

```diff
@@ -156,61 +156,119 @@ CSS to unitless numbers:
 
-CSS added support for [`min()` and `max()` functions] in Values and Units Level
-4, from where they were quickly adopted by Safari [to support the iPhoneX]. But
-Sass supported its own [`min()`] and [`max()`] functions long before this, and
-it needed to be backwards-compatible with all those existing stylesheets. This
-led to the need for extra-special syntactic cleverness.
-
-[`min()` and `max()` functions]: https://drafts.csswg.org/css-values-4/#calc-notation
+Sass parses the following functions as [calculations]:
+* Comparison Functions: [`min()`], [`max()`], and [`clamp()`]
+* Stepped Value Functions: [`round()`], [`mod()`], and [`rem()`].
+* Trigonometric Functions: [`sin()`], [`cos()`], [`tan()`], [`asin()`], [`acos()`], [`atan()`], and [`atan2()`].
+* Exponential Functions: [`pow()`], [`sqrt()`], [`hypot()`], [`log()`], and [`exp()`].
+* Sign-Related Functions: [`abs()`] and [`sign()`].
+
+[calculations]: https://www.w3.org/TR/css-values-4/#math
+[`min()`]: #min-and-max
+[`max()`]: #min-and-max
+[`clamp()`]: /documentation/modules/math/#clamp
+[`round()`]: #round
+[`sin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sin
+[`cos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/cos
+[`tan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/tan
+[`asin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/asin
+[`acos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/acos
+[`atan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan
+[`atan2()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan2
+[`pow()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/pow
+[`sqrt()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sqrt
+[`hypot()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/hypot
+[`log()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/log
+[`abs()`]: #abs
+[`exp()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/exp
+[`mod()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/mod
+[`rem()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/rem
+[`sign()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sign
+
+### Legacy Global Functions
+
+CSS added support for [mathematical expressions] in Values and Units Level
+4. However, Sass supported its own [`round()`], [`abs()`], [`min()`] and
+[`max()`] long before this, and it needed to be backwards-compatible with all
+those existing stylesheets. This led to the need for extra-special syntactic
+cleverness.
+
+[`min()` and `max()` functions]: https://www.w3.org/TR/css-values-4/#math
 [to support the iPhoneX]: https://webkit.org/blog/7929/designing-websites-for-iphone-x/
-[`min()`]: /documentation/modules/math#min
-[`max()`]: /documentation/modules/math#max
+[`round()`]: ../modules/math#round
+[`abs()`]: ../modules/math#abs
+[`min()`]: ../modules/math#min
+[`max()`]: ../modules/math#max
 
-If a `min()` or `max()` function call is a valid calculation expression, it will
+If a calculation function call is a valid calculation expression, it will
```

Comment:
I think it's worth specifically saying "a call to `round()`, `abs()`, `min()`, or `max()`" here, since this isn't true for _all_ calculations.